src/lib/renderComponent.tsx

```diff
@@ -221,6 +224,10 @@ const renderComponent = <P extends {}>(config: RenderConfig<P>): React.ReactElem
         const classes: ComponentSlotClasses = getClasses(renderer, mergedStyles, styleParam)
         classes.root = cx(className, classes.root, props.className)
 
+        const { children, content } = props
+        const rtlTransformedChildren = getRtlTransformedElement(children)
```

Comment:
it seems that this step is not necessary in quite majority of rendering cycles. What if we will just provide a function to the client component instead, so that RTL transform would be approached the following way:

```jsx
class SomeComponent extends UIComponent {
   renderComponent({ ..., addRtlSupport }) {   // just one option for naming, 
      const { children, content } = props
   
       <ElementType>
          {addRtlSupport(childrenExist(children) ? children : content)}
      </ElementType>
   }
}
```

Additional benefit is that we will eliminate ambiguity in choosing where `children` or `content` should be consumed from - as with current approach they should be consumed from props if not transformed, and from render args otherwise.